docs/api-reference/full-text/bm25.mdx

```diff
@@ -250,3 +250,43 @@ FROM <index_name>.search(
 <ParamField body="order_by_direction" default={"asc"}>
   The sort order to use.
 </ParamField>
+
+## Lenient Parsing
+
+Lenient parsing is disabled by default. To enable it, set the `lenient_parsing` parameter to `true`. This setting parses invalid query on a best effort basis.
```

Comment:
I realize that this is de-facto what Tantivy says in their docs... but the grammar is a little off, and I think we can be more helpful by specifically describing the parsing behavior. Maybe:

"The query parser will construct a valid query on a best-effort basis. The executed query will ignore invalid query components like missing or non-existent fields."